apps/test/tests-entry.js

```diff
@@ -0,0 +1,71 @@
+import '@babel/polyfill/noConflict';
```

Comment:
Also maybe a comment here for orientation?

Reply:
Love it, great point, we should definitely call out the special role of this file, esp that karma will use this file as the Official Webapck Entrypoint(TM).